qiskit_experiments/library/characterization/analysis/ramsey_xy_analysis.py

```diff
@@ -206,3 +209,293 @@ def _evaluate_quality(self, fit_data: curve.CurveFitResult) -> Union[str, None]:
+
+    # section: overview
+
+        This analysis is a variant of :class:`RamseyXYAnalysis` in which
+        the data is fit for a trigonometric function model with a linear phase.
+        By contrast, in this model, the phase is assumed to be a polynominal of the x-data,
+        and techniques to compute a good initial guess for these polynominal coefficients
+        are not trivial. For example, when the phase is a linear function of the x-data,
+        one may apply a Fourier transform to the data to estimate the coefficient,
+        but this technique can not be used for a higher order polynomial.
+
+        This analysis assumes the following polynominal for the phase imparted by the Stark shift.
+
+        .. math::
+
+            \theta_{\text Stark}(x) = 2 \pi t_S f_S(x),
+
+        where
+
+        .. math::
+
+            f_S(x) = c_1 x + c_2 x^2 + c_3 x^3 + f_{\rm err},
+
+        denotes the Stark shift. For the lowest order perturbative expansion of a single driven qubit,
+        the Stark shift is a quadratic function of :math:`x`, but linear and cubic terms
+        and a constant offset are also considered to account for
+        other effects, e.g. strong drive, collisions, TLS, and so forth,
+        and frequency mis-calibration, respectively.
+
+    # section: fit_model
+
+        .. math::
+
+            F_{X+} = \text{amp} \cdot \cos \left( r f_S^+(x) \right) + \text{offset}, \\
+            F_{Y+} = \text{amp} \cdot \sin \left( r f_S^+(x) \right) + \text{offset}, \\
+            F_{X-} = \text{amp} \cdot \cos \left( r f_S^-(x) \right) + \text{offset}, \\
+            F_{Y-} = \text{amp} \cdot \sin \left( r f_S^-(x) \right) + \text{offset},
+
+        where we put a constant :math:`r = 2 \pi t_S` and
+
+        .. math ::
+
+            f_S^\nu(x) = c_1^\nu x + c_2^\nu x^2 + c_3^\nu x^3 + f_{\rm err}.
+
+        The Stark shift is asymmetric with respect to :math:`x=0`, because of the
+        anti-crossings of higher energy levels. In a typical transmon qubit,
+        these levels appear only in :math:`f_S < 0` because of the negative anharmonicity.
+        To precisely fit the results, this analysis uses different model parameters
+        for positive (:math:`x > 0`) and negative (:math:`x < 0`) shift domains.
+
+        To obtain the initial guess, the following calculation is employed in this analysis.
+        First, oscillations in each quadrature are normalized and the offset is subtracted.
+        The amplitude and offset can be accurately estimated from the experiment data
+        when the oscillation involves multiple cycles.
+
+        .. math ::
+
+            {\cal F}_{X} = \cos \left( r f_S^+(x) \right), \\
+            {\cal F}_{Y} = \sin \left( r f_S^+(x) \right).
+
+        Next, these normalized oscillations are differentiated
+
+        .. math ::
+
+            \dot{{\cal F}}_X = r \frac{d}{dx} f_S \bar{{\cal F}}_Y, \\
+            \dot{{\cal F}}_Y = - r \frac{d}{dx} f_S \bar{{\cal F}}_X. \\
```

Comment:
Very minor since these get squared, but shouldn't the minus sign be on the other equation?

What does the `\bar` on the right mean?

Very minor as well, but I think `\frac{d f_S}{dx}` renders a little nice than `\frac{d}{dx} f_S` because that looks a little like the derivative operates on the product with F:

![image](https://github.com/Qiskit-Extensions/qiskit-experiments/assets/1178780/8b7d2fb1-604d-4986-8e9c-52b030b57c75)

Reply:
Thanks again for catching this. I replaced `\bar` with `\cal` to represent normalization but had forgotten to remove it. The minus sign comes from the derivative of cosine. Another one is derivative of sine and no minus sign should appear.

Reply:
All looks good -- sorry just one minor point -- shouldn't the minus sign be on `\dot{{ \cal F}}_X` since FX was the cosine case?

Reply:
oh feeling shame...